internal/jobobject/jobobject_test.go

```diff
@@ -65,6 +65,46 @@ func TestSiloCreateAndOpen(t *testing.T) {
 	}
 }
 
+func TestJobStats(t *testing.T) {
```

Comment:
Can we get a test to verify this panics/fails if `EnableIOTracking` isnt set?

Reply:
sure!

Reply:
Added!